plugin/src/main/java/io/spine/tools/bootstrap/JavaExtension.java

```diff
@@ -20,46 +20,136 @@
 
 package io.spine.tools.bootstrap;
```

Comment:
I think the package should be either `io.spine.tools.gradle.bootstrap` (under `gradle`) or `io.spine.tools.gradle` (assuming the reusability) with a small fraction of code residing under `gradle.bootstrap` for the very purpose of bootstrapping.